csharp/ql/src/Telemetry/SupportedExternalSinks.ql

```diff
@@ -0,0 +1,17 @@
+/**
+ * @name Supported sinks in external libraries
+ * @description A list of 3rd party APIs detected as sinks. Excludes test and generated code.
```

Comment:
Do we exclude generated code?

Reply:
We do not (copy paste error). Description will be updated to reflect this.